doc/migration/version-upgrade-guide.md

```diff
@@ -1103,6 +1103,68 @@ which ./bin/openapi-gen > /dev/null || go build -o ./bin/openapi-gen k8s.io/kube
                   -r "-"
 ```
 
+#### Molecule Upgrade for Ansible based-operators
+
+The Molecule version for Ansible based-operators was upgrade from `2.22` to `3.0.2`. Following the changes required in the default scaffold files.
+
+- Remove the `scenario.name` from molecule.yaml, and ensure that any condition with in the `verify.yaml` will look for the folder name which determines the scenario name from now on
```

Comment:
I'm not sure I understand this line, where is the scenario.name used within the verify.yml?

Reply:
Yep. I updated it .. good catch.